perfkitbenchmarker/providers/aws/aws_emr.py

```diff
@@ -95,6 +95,7 @@ class AwsEMR(spark_service.BaseSparkService):
   CLOUD = providers.AWS
   SPARK_SAMPLE_LOCATION = '/usr/lib/spark/lib/spark-examples.jar'
   SERVICE_NAME = 'emr'
+  cidr = None  # placeholder for vpn service
```

Comment:
Is there a plan to integrate here?

Reply:
No. EMR cluster passes itself to `aws_network.AwsNetwork.GetNetwork(self)` which expects zone and cidr attributes. This probably should have gone into [#1986] but didn't present problems until the aws network code was involved (this PR).